tests/mocha/server/HelpersTest.js

```diff
@@ -2,29 +2,26 @@ MochaWeb.testOnly(function() {
   describe('helpers', function() {
 
     describe('getCompetitionEvents', function() {
-      var compId = Competitions.insert({ competitionName: "Comp Etition", listed: false, startDate: new Date() });
+      beforeEach(function() {
+        this.compId = Competitions.insert({ competitionName: "Comp Etition", listed: false, startDate: new Date() });
```

Comment:
Yeah, I don't know where I copied that code from or why it works...

The problem with the earlier code is that if you throw an exception outside an `it`, the whole test suite silently dies.

It works without the `this.` part, but I settled on this:

    describe('getCompetitionEvents', function() {
      var compId;
      beforeEach(function() {
        compId = Competitions.insert({ competitionName: "Comp Etition", listed: false, startDate: new Date() });
      });